src/bin2llvmir/optimizations/dsm_generator/dsm_generator.cpp

```diff
@@ -225,7 +225,7 @@ void DsmGenerator::generateFunction(
 			<< fnc->getStart().toHexPrefixString()
 			<< " -- " << fnc->getEnd().toHexPrefixString() << "\n";
 
-	if (!fnc->isUserDefined())
+	if (!(fnc->isDecompilerDefined() || fnc->isUserDefined()))
```

Comment:
Consider moving the negation into the or-expression to make the code a bit more readable:
```suggestion
	if (!fnc->isDecompilerDefined() && !fnc->isUserDefined())
```

Reply:
Okay.